roles/networking_mapper/defaults/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+# All variables within this role should have a prefix of "cifmw_networking_mapper"
+cifmw_networking_mapper_basedir: >-
+  {{
+    cifmw_basedir | default(ansible_user_dir ~ '/ci-framework-data')
+  }}
+cifmw_networking_mapper_infra_dir: "/etc/ci/env"
+cifmw_networking_mapper_networking_def_path: "{{ cifmw_networking_mapper_infra_dir }}/networking-definition.yml"
```

Comment:
if you want to keep lines shorter, you may use the `path_join` filter. We should consider this (included myself) when we have long paths.